src/examples/deploy_application.yml

```diff
@@ -1,18 +1,18 @@
 description: >
-  Deploy an application to AWS CodeDeploy
+  Deploy an application to AWS CodeDeploy using static AWS keys for authentication
 usage:
   version: 2.1
   orbs:
     aws-code-deploy: circleci/aws-code-deploy@3.0
-    # import aws-cli orb to install and configure authentication
+    # Importing aws-cli orb is required for authentication
     aws-cli: circleci/aws-cli@3.1
 
   workflows:
     deploy_application:
       jobs:
         - aws-code-deploy/deploy:
-            pre-steps:
-              # install and configure aws credentials with static keys stored as env_vars
```

Comment:
comments are not captured in the usage examples on the CircleCI website, as we convert the YAML. Ensure any required notes are in the description